.semversioner/next-release/major-20240923234633073836.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type": "major",
```

Comment:
minor or patch, not major

Reply:
Recommend patch as it’s an update to an existing query strategy.